packages/storage/amplify_storage_s3/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0-next.6
+
+- Minor bug fixes and improvements
+
```

Comment:
question: Should any of the changes from dart package go here as well?

Reply:
Done 👍